lib/ts/BufferWriterFormat.cc

```diff
@@ -60,7 +60,26 @@ tv_to_positive_decimal(ts::TextView src, ts::TextView *out)
 
 namespace ts
 {
-const ts::BWFSpec ts::BWFSpec::DEFAULT;
+const BWFSpec BWFSpec::DEFAULT;
+const uint8_t BWFSpec::TYPE_FLAG[0x100] = {
```

Comment:
In writing formatters for this, checking for numeric vs. not was common and it seemed better to make a fast `ctype` style mechanism for doing that check, which is not only faster but harder to get wrong by forgetting one of the characters.